### ISSUE TITLE
"Custom SSL thumbprint" overrides "Accept all SSL certificates"

### ISSUE THREAD

**Issue by abugher** [erickok/transdroid#342]
My server has a new fingerprint, and for a moment I neglected to update transdroid, so of course it stopped working.  The error message only said there was a connection problem.  (I did not capture this.)  I realized my mistake, but before typing a long string, I decided to confirm that my out of date fingerprint was the only problem.  So in Transdroid, under the advanced settings for that server, I checked "Accept all SSL certificates", expecting that the connection would complete.

The interface allowed me to check the "accept all" option while a custom fingerprint is set, but Transdroid could not complete the connection until I removed the fingerprint.  It seems that if the fingerprint is set, that takes precedence over the "accept all" checkbox, and only the fingerprinted cert is accepted.

If that is the case, I would suggest that the "Accept all SSL certificates" option be disabled in a visually obvious way when the "Custom SSL thumbprint" is set.  The opposite would also be fine:  If "accept all" is checked, disable ("gray out") the "Custom SSL thumbprint" field.  That would actually be slightly better for my specific case, since I could toggle on and off strict SSL checking without removing the fingerprint from the settings.

**Comment by erickok** [comment]
Yes it is an interface glitch, although I think it makes sense that the better (more secure) feature overrides the other.
